src/libaktualizr/package_manager/packagemanagerinterface.cc

```diff
@@ -67,6 +72,15 @@ static int ProgressHandler(void* clientp, curl_off_t dltotal, curl_off_t dlnow,
   if (ds->progress_cb && progress > ds->last_progress) {
     ds->last_progress = progress;
     ds->progress_cb(ds->target, "Downloading", progress);
+    //OTA-4864:Improve binary file download progress logging. Report each XX sec report event that notify user
+    if (!ds->target.IsOstree()) {      
```

Comment:
This handler isn't used for OSTree, so you shouldn't need to use this check.

Reply:
done